doc/release-notes/release-1.4.1.md

```diff
@@ -0,0 +1,23 @@
+# khmer v1.4.1 release notes
+
+This is the v1.4.1 release of khmer. Due to the Python 3 migration we need to
```

Comment:
(You explain it below quite well, I note. But it needs to be expanded on here otherwise will derail)